canvas_modules/harness/src/client/App.js

```diff
@@ -1641,8 +1641,10 @@ export default class App extends React.Component {
 			{ action: "arrangeVertically", label: "Arrange Vertically", enable: true }
 		];
 
-		const notificationConfig = { action: "notification", label: "Notifications", enable: true, notificationHeader: "Notifications" };
-		const notificationConfig2 = { action: "notification", label: "Notifications", enable: true, notificationHeader: "Notifications Canvas 2" };
+		const notificationConfig = { action: "notification", label: "Notifications", notificationHeader: "Notification Center", notificationSubtitle: "subtitle status", enable: true,
+			emptyMessage: "You don't have any notifications right now.", clearAllMessage: "Clear all", keepOpen: true };
+		const notificationConfig2 = { action: "notification", label: "Notifications", notificationHeader: "Notification Center Canvas 2", enable: true,
+			emptyMessage: "You don't have any notifications right now.", clearAllMessage: "Clear all" };
 		const contextMenuConfig = {
```

Comment:
I know I commented on it before but ... let's make sure we document these new config options in the Wiki.
BTW - How does the user indicate they don't want the Clear All button?  Is it by ommiting that field from the config?

Reply:
Yes, if no "clear all" message is passed in, then no button will be shown. I will make sure to document this clearly in the wiki.